src/main/java/com/varbro/varbro/controller/finance/ExpenseController.java

```diff
@@ -30,6 +30,6 @@ public String financeAddExpenseSubmit(@Valid @ModelAttribute("expense") Expense
         if (!bindingResult.hasErrors()) {
             expenseService.saveExpense(expense);
         }
-        return "finance/add-expense";
+        return "finance/index";
```

Comment:
W porządku, tylko przydałaby się walidacja wprowadzonych danych i jakieś info o tym czy dodawanie poszło ok.

Reply:
Myślałem, żeby o dodaniu wydatku zostawać na tym widoku, żeby ułatwić dodawanie kolejnych, tylko trzeba by wtedy czyścić pola po dodaniu i generowac stosowny komunikat. Ale w zasadzie klient nie sprecyzował wymagań, a dział UX twierdzi, że można przekierowywać do panelu finansów. A co do walidacji to jest automatyczna przez thymeleaf i atrybuty modelu, ale jeszcze przydałaby się cienka na froncie, nie po stronie serwera.